converters/toZigbee.js

```diff
@@ -1248,6 +1248,32 @@ const converters = {
     // #endregion
 
     // #region Non-generic converters
+    ias_wd: {
+        key: ['max_duration'],
+        convertSet: async (entity, key, value, meta) => {
+            await entity.write('ssIasWd', {'maxDuration': value});
+            return {state: {max_duration: value}};
+        },
+        convertGet: async (entity, key, meta) => {
+            await entity.read('ssIasWd', ['maxDuration']);
+        },
+    },
+    warning_simple: {
+        key: ['alarm'],
+        convertSet: async (entity, key, value, meta) => {
+            const alarmState = (value === 'alarm' || value === 'OFF' ? 0 : 1);
```

Comment:
If I understand correctly, this converter accepts `OFF` and `alarm`, I think `OFF` and `START` better explains what it does.

Reply:
You want me to change this from ON / OFF to START/OFF ?

Reply:
I don't understand why  you check for `value` `alarm`, `const alarmState = value === 'OFF' ? 0 : 1;` should be good.

Reply:
Was leftover from earlier debugging :) Removed. PR already updated.